thirdparty/cmake_modules/koreader_thirdparty_common.cmake

```diff
@@ -5,7 +5,11 @@ if(NOT DEFINED PROCESSOR_COUNT)
     set(PROCESSOR_COUNT ${N})
 endif()
 
-set(ISED "sed -ie")
+if (DEFINED ENV{DARWIN})
+    set(ISED sed -i "" -e)
+else()
+    set(ISED sed -i -e)
+endif()
```

Comment:
Note: this addresses my comment in #1687.

Reply:
Except it does not work ;(, I'm thinking because the empty arguments is not passed.

Reply:
But `sed -ie` is not right, or at least is reported not to work in all macOS versions. (And is wrong with GNU sed when using multiple `-e` script too).

Reply:
Well, I'm perfectly fine leaving the ISED stuff as it was. ;-)

I was surprised to see this PR actually, given Meson. ^_^

Reply:
I fear I might have to resort to something like:
```cmake
set(ISED sh -c "sed -i '' -e \"$@\"" --)
```

Reply:
I assume you mean it's not some weird escaping issue like this?

```
set(ISED sed -i \"\" -e)
```

Reply:
Or perhaps even along these lines?

```
set(ISED "sed -i '' -e")
```

Reply:
> `set(ISED sed -i \"\" -e)`

No, the problem is, quoting the cmake documentation:
>  Each non-empty element is given to the command invocation as an argument.

> `set(ISED "sed -i '' -e")`

That's one argument.